packages/angular/package.json

```diff
@@ -53,6 +50,8 @@
     "@angular/platform-browser-dynamic": "11.0.0",
     "@angular/router": "11.0.0",
     "@cds/core": "./dist/core",
+    "@cds/angular": "./dist/angular",
+    "@clr/angular": "./dist/clr-angular",
```

Comment:
This ensures the dev apps in the angular project get the local copies when built.